Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/ProcessingNote.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataformat.access;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.kitodo.api.dataformat.mets.AgentXmlElementAccessInterface;
+import org.kitodo.dataformat.metskitodo.MetsType.MetsHdr.Agent;
+
+/**
+ * A processing note that can be placed in the header of the XML file.
+ */
+public class ProcessingNote implements AgentXmlElementAccessInterface {
+    /**
+     * Known roles in METS. If the role takes one of these values, the
+     * {@code ROLE} attribute is set to this value. Otherwise, the role is set
+     * to {@code OTHER} and the {@code OTHERROLE} attribute is set to the value
+     * of the role.
+     */
+    private static final List<String> KNOWN_ROLES = Arrays.asList("CREATOR", "EDITOR", "ARCHIVIST", "PRESERVATION",
```

Comment:
Seems like an Enum would be appropriate

Reply:
To check if a string matches a particular string? Not really, right?